sdk/servicebus/azure-servicebus/tests/async_tests/test_sessions_async.py

```diff
@@ -874,6 +874,9 @@ async def test_async_session_get_set_state_with_receiver(self, servicebus_namesp
                     count += 1
                 state = await receiver.session.get_state()
                 assert state == b'first_state'
+                await receiver.session.set_state(None, timeout=5)
+                state = await receiver.session.get_state()
+                assert not state
```

Comment:
Would also be good to clarify if a distinction between None and an empty string should be made here too.